packages/popover/src/Popover.story.tsx

```diff
@@ -57,10 +103,37 @@ const Template = ({ buttonText, ...args }: PopoverStoryProps) => {
   );
 };
 
-export const Basic = Template.bind({});
+export const ScrollableContainer = ({
+  refButtonPosition,
+  buttonText,
+  ...args
+}: PopoverStoryProps) => {
+  const [active, setActive] = useState<boolean>(false);
+  const [portalContainer, setPortalContainer] = useState<HTMLElement | null>(
+    null,
+  );
+
+  const position = referenceElPositions[refButtonPosition];
 
-export const ScrollableContainer = (args: PopoverStoryProps) => (
-  <div className={scrollableStyle}>
-    <Template {...args} />
-  </div>
-);
+  return (
+    <div className={scrollableStyle}>
+      <div className={scrollableInnerStyle} ref={el => setPortalContainer(el)}>
+        <button
+          onClick={() => setActive(active => !active)}
+          className={position}
+        >
+          {buttonText}
+          <Popover
+            {...args}
+            active={active}
+            usePortal={true}
+            portalContainer={portalContainer}
+            scrollContainer={portalContainer}
+          >
+            <div className={popoverStyle}>Popover content</div>
+          </Popover>
+        </button>
+      </div>
+    </div>
+  );
+};
```

Comment:
```
ScrollableContainer.args = {
  usePortal: true,
};
ScrollableContainer.argTypes = {
  usePortal: { control: 'none' },
}
```

Reply:
I was able to remove some controls but I kept running into an error when setting the default `usePortal: true,`

Reply:
If you're talking about the TS error, I think this one is safe to ignore with a `@ts-expect-error`